ocpp-v1_6/src/main/java/eu/chargetime/ocpp/feature/profile/ClientCoreProfile.java

```diff
@@ -187,11 +187,12 @@ public MeterValuesRequest createMeterValuesRequest(Integer connectorId, MeterVal
      * @see StartTransactionRequest
      * @see StartTransactionFeature
      */
-    public StartTransactionRequest createStartTransactionRequest(Integer connectorId, String idTag, Integer meterStart, Calendar timestamp) throws PropertyConstraintException {
+    public StartTransactionRequest createStartTransactionRequest(Integer connectorId, String idTag, Integer meterStart, int reservationId, Calendar timestamp) throws PropertyConstraintException {
```

Comment:
The idea with the create metodes on the profile, has to help make a request with all the mandatory fields set.
The field reservatopmID wasn't a part of this method, since it's not mandatory.
I'd recommend removing it from this method and setting it on the return object locally.